db/dbtypes/types.go

```diff
@@ -677,3 +680,198 @@ type SideChain struct {
 	Hashes  []string
 	Heights []int64
 }
+
+// AddressTx models data for transactions on the address page.
+type AddressTx struct {
```

Comment:
What is the motivation and justification for moving these types from `explorer` to `dbtypes`?  They really are designed to support the explorer rather than a db backend (AddressTx and AddressInfo are data for various pages, IOID does text formatting for display).

However, it always felt backwards that `dcrpg` should import `explorer`.

Reply:
Yeah. I sorta went out on a limb there. Initially, I wanted to use the code from `explorer.AddressPage` in a new function `explorer.AddressTable`. As I started to deconstruct it to move to what is now `pgblockchain.AddressData`, I saw that it was one of the few places in `db` where explorer types were being used, which as you mentioned felt backwards. 

I'm also thinking that there might be some additional optimization possible from a tighter integration with `pgblockchain`. For instance, there seems to be some code duplication here and here. 

It also seemed appropriate for the data source to do the brunt of the data manipulatio

Reply:
`dbtypes` works for me.  We'll just want to put a bit about this design change in the commit messages.

BTW, for 4.0 and such API changes, we'll be bumping the module version from v3 to v4 shortly.